napari/_qt/__init__.py

```diff
@@ -11,7 +11,7 @@
     if 'No Qt bindings could be found' in str(e):
         raise ImportError(
             trans._(
-                'No Qt bindings could be found.\n\nnapari requires either PyQt5 or PySide2 to be installed in the environment.\nTo install the default backend (currently PyQt5), run "pip install napari[all]" \nYou may also use "pip install napari[pyside2]"for Pyside2, or "pip install napari[pyqt5]" for PyQt5',
+                'No Qt bindings could be found.\n\nnapari requires either PyQt5 (default) or PySide2 to be installed in the environment.\n\nWith pip, you can install either with:\n  $ pip install -U \'napari[all]\'  # default choice\n  $ pip install -U \'napari[pyqt5]\'\n  $ pip install -U \'napari[pyside2]\'\n\nWith conda, you need to do:\n  $ conda install -c conda-forge pyqt\n  $ conda install -c conda-forge pyside2\n',
                 deferred=True,
```

Comment:
I love `from inspect import cleandoc` to improve readability of this stuff:

```suggestion
                cleandoc(
                """
                No Qt bindings could be found.

                napari requires either PyQt5 (default) or PySide2 to be installed in the environment.

                With pip, you can install either with:
                  $ pip install -U 'napari'  # default choice
                  $ pip install -U 'napari'
                  $ pip install -U 'napari'

                With conda, you need to do:
                  $ conda install -c conda-forge pyqt
                  $ conda install -c conda-forge pyside2
                """
                )
                deferred=True,
```

Reply:
I was under the impression that all translate strings have to be in a single line for some reason? There are plenty of examples there in the same module with equally long unreadable strings, so I am unsure whether we can clean it up?

Reply:
I vaguely remember translate+multiline strings being fixed...  #4588 I think? @Carreau probably can help more here.

Reply:
I believe that was fixed yes. ef0e0c52e7b4ce3e1195abe3d5e95acee8fc471b as part of #4588.

Reply:
So +1 to multiline.

Reply:
@jaimergp should we add some final line suggesting which one to use based on what we find?

For instance, if we find a `napari` package in the conda_meta, we can suggest conda, otherwise suggest pip

Or something like "It looks like napari was installed with conda" ? or similar or too manye weird corner cases to take care of?

Reply:
Thanks for the clarification. Accepted your suggestion, @brisvag. @goanpeca, I'll reply out-of-thread.